packages/devtools_app/lib/src/debugger/flutter/common.dart

```diff
@@ -46,17 +46,13 @@ Widget createAnimatedCircleWidget(double radius, Color color) {
   );
 }
 
-TextStyle regularStyle(BuildContext context) {
-  final theme = Theme.of(context);
-  return TextStyle(color: theme.textTheme.bodyText2.color);
-}
+extension DebuggerTextStyleExtension on BuildContext {
```

Comment:
Hmm, should these instead be defined on the ThemeData class? I think they're questions we're logically asking of the current theme. See the existing `ThemeDataExtension` extension - `Theme.of(context).isDarkTheme`.

Reply:
done